pep-0637.txt

```diff
@@ -140,6 +140,30 @@ specification would improve notation and provide additional value:
      >>> # new syntax
      >>> ds["empty"][lon=1:5, lat=6:] = 10
 
+6. Functions/methods whose arguments is another function (plus its
+   arguments) currently need to resort to ``functools.partial`` to
+   distinguish their keyword arguments from those of the passed-in
+   function. [#trio-run]_
+
+   An indexed notation would afford a Pythonic way to pass keyword
+   arguments to these functions without cluttering the caller's code.
+
+   ::
+
+     >>> # syntax without keywords. `name=` is processed by `trio.run`.
+     >>> trio.run(func, param1, param2, name="func")
```

Comment:
In the other examples, param2 takes a value. This one doesn't. Is that right?

Reply:
I changed those to `value1` and `value2` which hopefully should clear this up.